docs/applications/configuration-management/getting-started-with-ansible/index.md

```diff
@@ -0,0 +1,240 @@
+
+* **Tasks**: The simplest unit of execution in Ansible is a task. Tasks utilize Ansible modules to manage your host's services, packages, files and to perform various system configurations. Tasks can be executed from the command line or within Playbooks.
+
+* **Playbooks**: Playbooks are YAML files containing a list of tasks in the desired order of execution. You can run Playbooks on your managed nodes and reuse and share them. [Variables](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html) and [Jinja templating](https://docs.ansible.com/ansible/latest/user_guide/playbooks_templating.html) provide a powerful way to execute complex tasks on your managed hosts.
+{{</ disclosure-note >}}
+
+## Scope of this Guide
+This guide introduces the basics of installing Ansible and preparing your environment to use [Playbooks](https://docs.ansible.com/ansible/latest/user_guide/playbooks.html). You will complete the following steps in this guide:
+
+* Install and configure Ansible on your computer or a Linode to serve as the control node that will manage your infrastructure nodes.
+* Create two Linodes to manage with Ansible and establish a basic connection between the control node and your managed nodes. The managed nodes will be referred to as `node-1`, and `node-2` throughout the guide.
+
+    {{< note >}}
+The examples in this guide provide a manual method to establish a basic connection between your control node and managed nodes as a way to introduce the basics of Ansible. If you would like to learn how to use Ansible's [Linode module](https://docs.ansible.com/ansible/latest/modules/linode_v4_module.html) to automate deploying and managing Linodes, see the [How to use the Linode Ansible Module to Deploy Linodes](/docs/applications/configuration-management/deploy-linodes-using-ansible/). The guide assumes familiarity with Ansible modules, Playbooks, and dynamic inventories.
+    {{</ note >}}
+
+## Before You Begin
+
+{{< caution >}}
+This guide's example instructions will create up to three billable Linodes on your account. If you do not want to keep using the example Linodes that you create, be sure to [delete them](#delete-a-cluster) when you have finished the guide.
+
+If you remove the resources afterward, you will only be billed for the hour(s) that the resources were present on your account. Consult the [Billing and Payments](/docs/platform/billing-and-support/billing-and-payments/) guide for detailed information about how hourly billing works and for a table of plan pricing.
+{{< /caution >}}
+
+1. [Create three Linodes](/docs/getting-started/#create-a-linode) running Debian 9. One will be the **control node** and two you will use as your Ansible **managed nodes**. The examples in this guide can also be followed using a single managed node, if preferred.
```

Comment:
Here it said just to create 3 and then they would all be managed nodes, which is not quite true. Only 2 would be used for managed nodes, named node-1 and node-2, as stated above. So I clarified that.